lighthouse-core/report/v2/templates.html

```diff
@@ -454,6 +454,7 @@ <h1 class="leftnav__header__title">Lighthouse</h1>
       font-size: var(--body-font-size);
       line-height: var(--body-line-height);
       margin-top: calc(0.5 * var(--body-line-height));
+      text-align: center;
```

Comment:
`align-items` will center the container but the text inside is still left aligned. It's only noticeable when the text wraps. Renaming to `SEO` will avoid the wrapping for now, but adding more categories or including long names will run into this problem again so I think it's necessary.